src/js/edu-benefits/1990-rjsf/BenefitsRelinquishmentField.jsx

```diff
@@ -84,8 +84,33 @@ export default class BenefitsRelinquishmentField extends React.Component {
       errorSchema,
       idSchema,
       onBlur,
-      registry
+      onChange,
+      registry,
+      required,
+      name,
+      formContext
     } = this.props;
+
+    const benefitsRelinquished = _.get('benefitsRelinquished', formData);
+    if (formContext.reviewMode &&
```

Comment:
I'm not a huge fan of adding these conditionals, but it's been the easiest way so far for custom fields (vs custom widgets, where we can use `reviewWidget`)